cmd/zetae2e/local/local.go

```diff
@@ -415,11 +413,20 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 			e2etests.TestSPLDepositName,
 			e2etests.TestSPLDepositAndCallName,
 			e2etests.TestSPLWithdrawName,
-			e2etests.TestSPLWithdrawAndCallName,
-			e2etests.TestSPLWithdrawAndCallRevertName,
 			e2etests.TestSPLWithdrawAndCreateReceiverAtaName,
 			e2etests.TestSolanaWhitelistSPLName,
 		}
+
+		// Add newly added tests(v29) if not running an upgrade
+		if !deployerRunner.IsRunningUpgrade() {
+			solanaTests = append(solanaTests, []string{
+				e2etests.TestSolanaWithdrawAndCallName,
```

Comment:
You should just put a `MinimumVersion` guard on the incompatible tests

Reply:
https://github.com/zeta-chain/node/pull/3601#discussion_r1973992023
Sems like the newer contracts are available now , we dont need this check